server/routes/objects.js

```diff
@@ -246,52 +239,41 @@ router.post('/:url', async function (req, res) {
         let tempName = parts[0];
         let inProgress = req.query.inProgress;
         let socket_id = req.query.socket_id;
-        let fileId = uuidv4();
         let pngFileName = tempName + '.png';
         let tempDirPath = path.resolve(__dirname, '../temp');
```

Comment:
The error handling for sharp errors should log the error using the logger instead of console.error. This ensures consistent logging throughout the application.
```diff
-                console.error(`sharp error:  ${err}`);
+                log.error(`sharp error:  ${err}`);
```
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*